data_models/data_refinery_models/models/test_jobs.py

```diff
@@ -0,0 +1,65 @@
+from django.test import TestCase
+from data_refinery_models.models import (
+    SurveyJob,
+    DownloaderJob,
+    ProcessorJob,
+    Batch,
+    BatchStatuses
+)
+
+
+def get_batch():
+    survey_job = SurveyJob(source_type="ARRAY_EXPRESS")
+    survey_job.save()
+
+    batch = Batch(
+        survey_job=survey_job,
+        source_type="ARRAY_EXPRESS",
+        size_in_bytes=0,
+        download_url="example.com",
+        raw_format="CEL",
+        processed_format="PCL",
+        pipeline_required="AFFY_TO_PCL",
+        platform_accession_code="A-AFFY-1",
+        experiment_accession_code="E-MTAB-3050",
+        experiment_title="It doesn't really matter.",
+        name="CE1234.CEL",
+        internal_location="A-AFFY-1/AFFY_TO_PCL/",
+        organism_id=9606,
+        organism_name="HOMO SAPIENS",
+        release_date="2017-05-05",
+        last_uploaded_date="2017-05-05",
+        status=BatchStatuses.NEW.value
+    )
+    batch.save()
+    return batch
+
+
+class DownloaderJobTestCase(TestCase):
+    def test_create_job_and_relationships(self):
+        """DownloaderJob, Batches, and relationships are created."""
+        batches = [get_batch(), get_batch()]
```

Comment:
I think I might disagree. The point of having two batches here is to make sure the code handles more than one well, it doesn't really do anything with any of the values so if any of them were different I don't see what it would change. If you're worried about the batches being compared at some point and potentially that comparison returning that they're equal, I don' think that's an issue because `get_batch` saves the batches which will give them an id which won't be the same between the two.